src/Cedar/Session.c

```diff
@@ -1261,21 +1261,10 @@ void StopSessionEx(SESSION *s, bool no_wait)
 	// Event
 	Set(s->HaltEvent);
 
-	if (s->ServerMode == false)
-	{
-		// Client mode
-		if (s->Connection)
-		{
-			StopConnection(s->Connection, no_wait);
-		}
-	}
-	else
+	// both Server and Client modes

```

Comment:
I would change the comment to:
```
// Server and client mode
```

Reply:
"modes" ?

Reply:
I think that `modes` makes it sound like as if there were multiple client/server modes.